src/main/java/com/joss/conductor/mobile/Conductor.java

```diff
@@ -77,13 +77,19 @@
      * @return The implementing class for fluency
      */
     Test swipeCenter(SwipeElementDirection direction);
+    Test swipeCenter(SwipeElementDirection direction, int swipeDurationInMillis);
+    Test swipeCenter(SwipeElementDirection direction, short numberOfSwipes);
```

Comment:
Seems like it would be very easy to call the wrong overload here. I'd go with either one or the other. Or maybe make a builder.

Reply:
Good call. Removed the methods with a signature of only `short numberOfSwipes`.